meson.build

```diff
@@ -44,6 +44,37 @@ if get_option('enable-tflite')
   cdata.set('HAVE_TFLITE', 1)
 endif
 
+# Define library dependencies for TensorRT support
+if get_option('enable-tensorrt')
+
+  # CUDA versions supported
+  cuda_versions = [
+    '10.0',
+  ]
+
+  # Find CUDA version
+  cuda_ver = ''
+  foreach v : cuda_versions
+    if cuda_ver == ''
+      cuda_dep = dependency('cudart-' + v, required: false)
+      cudart_dep = dependency('cudart-' + v, required: false)
+      if cuda_dep.found() and cudart_dep.found()
+        cuda_ver = v
+      endif
+    endif
+  endforeach
+
+  if not cuda_dep.found() or not cudart_dep.found()
+    error ('Unable to find CUDA supported version')
+  endif
+
+  tensorrt = cpp.find_library('nvinfer', required: true)
+  tensorrt_dep = declare_dependency(dependencies: tensorrt)
+  cuda_inc_dir = cuda_dep.get_pkgconfig_variable('includedir')
```

Comment:
This should be unnecessary. This is a symptom of a dependency not carried appropriately.